lgc/interface/lgc/Pipeline.h

```diff
@@ -228,7 +228,8 @@ struct ResourceNode {
       unsigned set;                   // Descriptor set
       unsigned binding;               // Binding
       unsigned stride;                // Size of each descriptor in the indexable range in dwords.
-      llvm::Constant *immutableValue; // Array of vectors of i32 constants for immutable value
+      unsigned immutableSize;         // Size (in units of DescriptorSizeSampler dwords) of immutableValue array
+      const uint32_t *immutableValue; // Array of dwords for immutable sampler.
```

Comment:
uint32_t -> unsigned

Reply:
I deliberately made this uint32_t because it is specifically a 32-bit value, rather than a general int value.